lib/lcset.gi

```diff
@@ -0,0 +1,139 @@
+##############################################################################
+##
+#W  lcset.gi                 GAP4 package `Utils'                Chris Wensley
+##
+#Y  Copyright (C) 2015-2022, The GAP Group 
+
+#############################################################################
+##
+#O  LeftCoset( <g>, <U> ) . . . . . . . . forms a left coset of U by g \in G 
+##
+InstallMethod( LeftCoset, "generic method for left cosets", true, 
+    [ IsObject, IsGroup ], 0,
+function( g, U )
+    local lc;
+
+    lc := rec();
+    ObjectifyWithAttributes( lc, IsLeftCosetType,
+        Representative, g,
+        ActingDomain, U, 
+        IsLeftCoset, true );
```

Comment:
I would expect `IsLeftCoset` to be already implied by `IsLeftCosetType`

Reply:
see later comment.